src/viper/syntax.ml

```diff
@@ -103,4 +103,3 @@ and typ' =
   | TupleT
   | OptionT of typ
   | ConT of id * typ list
-
```

Comment:
Please avoid unnecessary formatting changes (especially the trailing newline, I think we were asked to keep it)

Reply:
Just out of curiosity. Shall we keep 2 trailing newlines at the end of the file?